src/main/java/ch/njol/skript/effects/EffSendTitle.java

```diff
@@ -84,19 +87,38 @@ public boolean init(final Expression<?>[] exprs, final int matchedPattern, final
 	@SuppressWarnings("null")
 	@Override
 	protected void execute(final Event e) {
-		String title = this.title != null ? this.title.getSingle(e) : "",
-		sub = subtitle != null ? subtitle.getSingle(e) : null;
+		String tit = title != null ? title.getSingle(e) : null;
```

Comment:
Before my changes, the names of variables were

(`Expression` -> `int`/`String`)
`title` -> `title`
`subtitle` -> `sub`
`fadeIn` -> `in`
`stay` -> `stay`
`fadeOut` -> `out`

After my changes it is:

`title` -> `tit`
`subtitle` -> `sub`
`fadeIn` -> `in`
`stay` -> `st`
`fadeOut` -> `out`

So, I tried to make it consistent and that's why I've given shortened names to all of these variables, not just some of them. I think it's more clearer that way. Everyone looking at the code should know what "st" is, because its scope is only the next 20 lines. But maybe something like exprTitle, exprSubtitle, exprFadeIn etc. would be better for `Expression` vars, and then int/String vars would be title, subtitle, fadeIn etc.

Reply:
You don't need different names for fields and variables

Reply:
Also, shortening more often used fields/variables should stay meaningful, `in` by itself is a preposition and meaningful enough in its context but `st` isn't, same goes for `tit`

Reply:
@TPGamesNL So maybe I'll name them all the same?

Reply:
Ok, so I named the strings/ints the same as expressions  and it looks nice.